jenkins/image_building/dib_elements/ci-base/pkg-map

```diff
@@ -21,14 +23,17 @@
           "qemu-kvm":"qemu-kvm",
           "tree": "tree",
           "vim": "vim-enhanced",
+          "virt-manager": "virt-manager",
           "wget": "wget"
         },
         "debian":{
           "bash-completion":"bash-completion",
+          "build-essential":"build-essential",
           "curl": "curl",
           "dnsmasq":"dnsmasq",
           "git": "git",
           "libguestfs-tools":"libguestfs-tools",
+          "libvirt-daemon-system":"",
```

Comment:
Why we're adding this in the first place, since on both redhat and debian it is mapped to `""` ?

Reply:
Oh I missed that. Why is it mapper to ""?

Reply:
fixed